lib/local-storage-adapter.js

```diff
@@ -20,7 +19,7 @@ function createLocalStorageAdapter (queueName) {
   var setPending = false
   var queueCache = []
 
-  var flushOnDefer = _.debounce(flush, 0)
+  var flushOnDefer = onceOnDefer(flush)
```

Comment:
Okay, this should have the same functionality now and I added a test that caught the error.